service-front/module/Application/src/View/Helper/LocalViewRenderer.php

```diff
@@ -0,0 +1,31 @@
+<?php
+// @inheritdoc
+//
+
+namespace Application\View\Helper;
+
+use Twig\Environment as TwigEnvironment;
+
+class LocalViewRenderer implements RendererInterface
+{
+     // this wraps TwigWrapper because TwigWrapper cannot be mocked due to being marked final
+     // in other words some acrobatics to get around incompatibility between Twig and Mockery
+
+    /**
+     * @param TwigEnvironment $viewRenderer
+     */
+    public function __construct(TwigEnvironment $viewRenderer)
+    {
+        $this->viewRenderer = $viewRenderer;
+    }
+
+    public function LoadTemplate(string $templateName)
+    {
+         return $this->viewRenderer->load($templateName)->unwrap();
+    }
+}
+
+interface RendererInterface
```

Comment:
split into separate file possibly - add autodocs here and use @inheritdoc on the class implementing this interface.

Reply:
Done